CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Fixes
+* **Fix issue with browser caching by default** [issue/38](https://github.com/cloudflare/kv-asset-handler/issues/38)
+  *  Because the edge cache was caching a `Cache-Control` header of 100 days. This fix strips out any `Cache-Control` header is `browserTTL` is not explicitly set. Also sets default edge cache max-age to 2 days. 
```

Comment:
**Don't use browser cache by default** 
- Previously, `kv-asset-handler` was returning a `Cache-Control` header of 100 days to the browser. After this fix, the `Cache-Control` header will be absent if `options.cacheControl.browserTTL` is not explicitly set. 

**Set default edge caching to 2 days**
- Previously the default cache time for static assets was 100 days, it is now 2 days. This can be overridden with `options.cacheControl.edgeTTL`